s3/deployment/src/test/java/io/quarkus/amazon/s3/deployment/S3CustomCredentialsProviderMissedBeanNameTest.java

```diff
@@ -3,13 +3,20 @@
 import static org.hamcrest.MatcherAssert.*;
 import static org.hamcrest.Matchers.equalTo;
 
+import jakarta.enterprise.inject.Instance;
+import jakarta.inject.Inject;
+
 import org.junit.jupiter.api.*;
 import org.junit.jupiter.api.extension.*;
 
 import io.quarkus.test.QuarkusUnitTest;
+import software.amazon.awssdk.services.s3.S3Client;
 
 class S3CustomCredentialsProviderMissedBeanNameTest {
 
+    @Inject
```

Comment:
the test was failing without injecting a client instance. Need to investigate if this is expected to have valid clientBuilder without client injectionPoint.